portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase

```diff
@@ -0,0 +1,120 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Verify that a form can be submited with fields in two different languages,"
+	@priority = 5
+	test CheckIsPossibleSubmitFormWithTwoDifferentLanguages {
+		task ("Add a form with a text field defined as Full Name") {
+			FormsAdmin.addForm();
+
+			Form.editName(formName = "New Forms");
+
+			Form.gotoAddField(
+				fieldPositionNumber = 0,
+				fieldType = "Text");
+
+			Form.editFieldLabelMetal(fieldValue = "Full Name");
+
+			Form.closeSidebar();
+		}
+
+		task ("Add a numerical field defined as Age") {
+			Form.gotoAddField(
+				fieldPositionNumber = 1,
+				fieldType = "Numeric");
+
+			Form.editFieldLabelMetal(fieldValue = "Age");
+		}
+
+		task ("Change the language to portuguese, add* new values and submit again") {
+			Form.addTranslation(localizedLanguage = "Portuguese (Brazil)");
+
+			FormViewBuilder.editFieldLabel(
+				fieldLabel = "Full Name",
+				label = "Nome Completo");
+
+			FormViewBuilder.editFieldLabel(
+				fieldLabel = "Age",
+				label = "Idade");
+		}
+
+		task ("We publish the form, save the form, and navigate to the published form") {
+			Form.publishForm();
+
+			Form.save();
+
+			FormsAdminNavigator.gotoPublishedForm();
+		}
+
+		task ("Aentering values into the form and sending the entered values") {
+			FormViewBuilder.editText(
+				fieldName = "Full Name",
+				fieldValue = "Henrique");
+
+			FormViewBuilder.editText(
+				fieldName = "Age",
+				fieldValue = 23);
+
+			FormPortlet.submit();
```

Comment:
- Use just FormPortlet.submitSuccessfully();, this way you ensure that the form has been submitted